registrations/management/commands/fix_pmtct_registrations.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand
+
+from registrations.models import Registration
+from registrations.tasks import validate_subscribe
+
+
+class Command(BaseCommand):
+    help = ("Find and fixes all PMTCT registrations that aren't validated"
+            "because of the edd date not being populated")
+
+    def handle(self, *args, **kwargs):
+
+        registrations = Registration.objects.filter(
+            validated=False,
+            reg_type__in=("pmtct_prebirth", "pmtct_postbirth")).iterator()
+
+        updates = 0
+
+        for registration in registrations:
+
+            related_regs = Registration.objects.filter(
+                    validated=True,
+                    registrant_id=registration.registrant_id
+                ).exclude(reg_type__in=("pmtct_prebirth", "pmtct_postbirth")).\
+                order_by('-created_at')
+
+            for related_reg in related_regs:
+
+                if related_reg.data.get("edd"):
+                    del registration.data["invalid_fields"]
```

Comment:
I'm wondering if there's a case where the `invalid_fields` field does not exist, but the registration still doesn't have the edd field. Maybe if the celery task crashed early or something. Maybe we should do a `registration.data.pop('invalid_fields', None)` or wrap it in a try catch so that this doesn't crash if the field doesn't exist.